opm/input/eclipse/EclipseState/Grid/NNC.hpp

```diff
@@ -63,6 +64,10 @@ struct NNCdata {
     size_t cell1;
     size_t cell2;
     double trans;
+    /// \brief if true the NNC is from the data file, otherwise created by PINCH
+    ///
+    /// For PINCH option(4) ALL we use this to overwrite transmissibilities.
+    char notFromPinch{1};
```

Comment:
@akva2 Up to now I do not check this in `operator==`. Should I for the serialization? For other reasons I am not so sure.

An alternative would be to skip the marker and save the PINCH NNCs separately, Maybe that would be cleaner? :thinking:

Reply:
if it is serialized and transferred to other processes, it should also be in the comparison operator and it should be given a non-default value in the serializationTestObject() method.

Reply:
done